targets/unity-v2/Testing/Tests/Shared/Uunit/UUnitIncrementalTestRunner.cs

```diff
@@ -97,9 +97,17 @@ private void OnLoginSuccess(LoginResult result)
                 GeneratePlayStreamEvent = true
             };
 
+            SaveReportToLocalWorkspace(suite.GetInternalReport());
+
             clientInstance.ExecuteCloudScript(request, OnCloudScriptSubmit, OnPostTestResultsError, null, testTitleData.extraHeaders);
         }
 
+        private void SaveReportToLocalWorkspace(TestSuiteReport testSuiteReport)
+        {
+            // TODO: this should put the file under our workspace directory
+            JUnitXml.WriteXmlFile(Directory.GetCurrentDirectory() + "..\\..\\..\\test.xml", new List<TestSuiteReport>() { testSuiteReport }, true);
```

Comment:
Not sure what to do about this path combination.  It looks super fragile.  Maybe consider defining a higher level root as an anchor like a workspace environment variable.

Reply:
We will address this issue on Monday.